.github/workflows/python-test-python-version.yml

```
@@ -0,0 +1,57 @@
+on:
+  workflow_call:
+
+defaults:
+  run:
+    working-directory: ./python
+
+jobs:
+  python_tests:
+    name: Python ${{ matrix.py_version }} Unit Tests
+    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        py_version: ["3.9.13"]   # py_version: ["3.9.13", "3.10.5", "3.11.7"]
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.py_version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.py_version }}
+
+    - name: Install the AaC Specification Package
+      run: pip install -r requirements.txt
```

```suggestion
      run: pip install -e .
```

I don't think swapping to an install of just the requirements gets what was desired of a package install.